docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,462 @@
+
+Next.js automatically caches the rendered result of React Components on the server at build time. This is an optimization that allows you to skip rendering components on the server on every request, and instead directly send the cached result to the client, resulting in faster page loads.
+
+> **Related terms**:
+>
+> - The terms **Automatic Static Optimization**, **Static Site Generation**, or **Static Rendering** might be also be used interchangeably to refer to the process of rendering and caching parts of your application code on the server at build time.
+
+To understand how the Render Cache works, it's helpful to look at how React handles rendering, and how Next.js caches the result:
+
+#### 1. React Rendering on the Server
+
+On the server, Next.js uses React's APIs to orchestrate rendering. This process is split into three stages:
+
+1. React renders Server Components into a special data format, optimized for streaming, called the **React Server Component Payload**.
+2. Next.js compiles, bundles, and code-splits the **JavaScript instructions** for rendering Client Components on the client.
+3. Next.js uses the React Payload and JavaScript instructions to pre-render **HTML** on the server.
+
+> **What is React Server Component Payload?**
+>
+> The React Payload is a compact string representation of the React Component tree. It's used by React on the client to update the DOM. The React Payload contains:
+>
+> - The rendered result of Server Components
+> - Placeholders for where Client Components should be rendered on the client
+> - Any serialized props passed to these Client Components
+> - References to the JavaScript instructions needed to render the Client Components
+> - Metadata tags, such as references to stylesheets
+>
+> To learn more, see the [React Rendering](/docs/app/building-your-application/rendering/react-components) documentation.
+
+#### 2. Next.js Caching on the Server (Render Cache)
+
+On the server, at build time, Next.js caches the React Server Component Payload, Client Component JavaScript, and pre-rendered HTML.
+
+#### 3. Next.js Caching on the Client (Router Cache)
+
+At request time, the React Server Component Payload is stored in the client-side [Router Cache](#router-cache) - a separate in-memory cache, split by individual routes. This Router Cache is used to improve the navigation experience by prefetching.
+
+#### 4. React Hydration and Reconciliation on the Client
+
+Then, on the client:
+
+1. The HTML is used to show an initial preview for Client and Server Components.
+2. The React payload is used to reconcile the Client and Server Component trees, and update the DOM.
+3. The JavaScript instructions are used to [hydrate](https://react.dev/reference/react-dom/client/hydrateRoot) Client Components.
+
+#### 5. Subsequent Navigations
+
+On subsequent navigations or during prefetching, Next.js will check if the React Payload is stored in Router Cache. If so, it will skip sending a new request to the server.
```

Comment:
```suggestion
On subsequent navigations or during prefetching, Next.js will check if the React Server Components Payload is stored in Router Cache. If so, it will skip sending a new request to the server.
```